### ISSUE TITLE
Automation of Metallurgic Infuser issue..

### ISSUE THREAD

**Issue by CloudConstructor1** [mekanism/Mekanism#6843]
#### Issue description:
I am playing on the latest Direwolf20 1.16 mod pack. I tried multiple ways of trying to push in enriched redstone, diamond and obsidian into the Infuser using logistic pipes, AE2 and Refined storage. Within the infuser sides config tabs, the items no long have the infusion input selection which was a yellow(pic#1) box in the 1.16.3-10.0.15,441. Now with that selection gone I figured the infusion tab was working. I have all sides "red/input" (pic#2). As you can see in (pic#3), pipes wont even connect. I can manually put them in though. In order to get them to work within the Direwolf20 pack, I had to rev down to version, 1.16.3-10.0.15,441. I also reported the issue to the mod pack author but since I think its mod specific theres probably nothing he can do. At least he knows though. 

PIC#1:
Version 1.16.3-10.0.15,441-Used only for screen shot to show yellow box.
![2021-01-04_14 25 31](https://user-images.githubusercontent.com/24704742/103576778-48023880-4e99-11eb-8894-d33b9f979e0d.png)

PIC#2:
![2021-01-04_14 24 26](https://user-images.githubusercontent.com/24704742/103577061-b9da8200-4e99-11eb-8e4b-8ec84cd297c4.png)

PIC#3:
![2021-01-04_14 25 05](https://user-images.githubusercontent.com/24704742/103577157-e1314f00-4e99-11eb-970f-807adb95449c.png)

#### Steps to reproduce:
Using Enriched Redstone, Enriched Diamond, and Enriched Obsidian
1. Place infuser.
2. Open sides config.
3. Click on Items Tab, turn off inputs.
4. Click on Infusion Tab
5. Open all sides to input.
6. Place Logistical Pipes, Refined Storage exporters, AE2 interfaces. 
7. Pipes wont even connect, and digital storage mods wont push into any side.
8. Installed version 1.16.3-10.0.15,441 of Mek, Mek Gen, and Mek Tools for some sort of automation again.

#### Version (make sure you are on the latest version before reporting):
**Forge: 35.1.15
**Mekanism: Mekanism-1.16.4-10.0.18.445 & .444
**Other relevant version: DireWolf20 1.16 mod pack V1.2

**Comment by pupnewfster** [comment]
If you look closely at your pictures, the first one you correctly have the **items** tab set to *extra*, whereas in the second picture you are looking at the **infuse types** tab. This other tab is for the chemical version of infuse types, setting a side to output, and then using a pressurized tube in extract mode allows for forcefully extracting said infuse type if you want to say mass store/only have a single input point for where you convert items to infuse types. The issue you are running into here is that you are trying to insert items, so need to be configuring the **items** tab, and set the side config to the yellow *extra* setting.

**Comment by CloudConstructor1** [comment]
Then there is a problem. The first pic was from an older version then the one currently in the DW20 pack. As stated, that pic was used only as a screen shot to describe what I was reffering to. And within my description I said that selection option for EXTRA input was no longer available in the new 1.16.4 versions of mekanism. Therefore i can tell the machine what side to input the extras into.

**Comment by CloudConstructor1** [comment]
Id suggest actually playing the mod pack if you think im lying.

**Comment by pupnewfster** [comment]
Is the second picture still valid? If so try actually reading my response, if not why did you include it?

**Comment by CloudConstructor1** [comment]
I included that first pic because I was trying to describe the yellow button since I wasn't getting that option in the 1.16.4 versions. I just completely deleted the instance and reinstalled it. Now, that selection is there. LOL, I know it wasn't there when I spent an hr or so gathering screenshots, information and typing everything out. Guess I should have done clean install. Maybe something corrupted on the last update. Sorry for your time.
